standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/StatObjectConverter.java

```diff
@@ -525,6 +531,16 @@ public static MPartitionColumnStatistics convertToMPartitionColumnStatistics(
     return mColStats;
   }
 
+  private static byte[] getBitVector(byte[] bytes) {
```

Comment:
you could move this logic into `MTableColumnStatistics#getBitVector` ;

Reply:
done